packages/hir/src/lowering/declarations.candy

```diff
@@ -342,12 +340,12 @@ impl HirClass: HasAst<AstClass> & HasDeclarations {
 
   fun astDeclarations(
```

Comment:
Is this function still necessary, or can we remove this function and just return an empty list from `typeUnsafeAstDeclarations(…)` below?